modules/javafx.graphics/src/main/java/javafx/scene/AccessibleAttribute.java

```diff
@@ -337,6 +337,25 @@ public enum AccessibleAttribute {
      */
     INDETERMINATE(Boolean.class),
 
+    /**
+     * Returns toggle state of CheckBox of CheckBoxTreeItem.
+     * <ul>
+     * <li>Used by: CheckBoxTreeItem</li>
+     * <li>Needs notify: yes </li>
+     * <li>Return Type: {@link Integer}
+     *   <ul>
+     *    <li>2: Indeterminate state</li>
+     *    <li>1: Checked state</li>
+     *    <li>0: Unchecked state</li>
+     *   </ul>
+     * </li>
+     * <li>Parameters: </li>
+     * </ul>
+     *
+     * @since 21
+     */
+    TOGGLE_STATE(Integer.class),
```

Comment:
Would a dedicated enum be a better choice here?

Reply:
Added a new enum `ToggleState`.
It is added as a member of `AccessibleAttribute` : Please see if it is a good place for the enum.